test/FunctionalTests/Microsoft.OData.Client.Tests/ALinq/SequenceMethodsTests.cs

```diff
@@ -0,0 +1,209 @@
+//---------------------------------------------------------------------
+// <copyright file="SequenceMethodsTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.OData.Edm;
+using Xunit;
+
+namespace Microsoft.OData.Client.Tests.ALinq
+{
+    /// <summary>
+    /// Tests to check sequence methods support.
+    /// </summary>
+    public class SequenceMethodsTests
+    {
+        private readonly TestContext _ctx;
+
+        public SequenceMethodsTests()
+        {
+            _ctx = new TestContext();
+        }
+
+        [Fact]
+        public void Any()
+        {
+            _ctx.InterceptRequestAndAssertUri("/Customers/$count");
+            _ctx.InterceptRequestAndMockResponse("91");
+            bool exists1 = _ctx.Customers.Any();
+            Assert.True(exists1);
```

Comment:
nitpick, but I'd suggest more descriptive variable names when possible
```suggestion
            bool hasCustomers = _ctx.Customers.Any();
            Assert.True(hasCustomers);
```